source/MaterialXContrib/MaterialXMaya/README.md

```diff
@@ -116,6 +152,20 @@ optionVar -stringValue materialXEditorPath "D:/MyEditor/bin/MyEditor.exe"
 
 To open the MaterialX document in the editor, press **Edit** in the Attribute Editor of the MaterialX node. Once the document has been edited and saved, press **Reload** in the Attribute Editor which executes the `reloadMaterialXNode` command and refreshes the shader in the viewport.
 
+## Debugging
```

Comment:
Another new section.

Reply:
May I suggest a slight rewording of the 2nd sentence:
"The Maya API allows plug-ins to configure paths to which these intermediate files can be saved for debugging purposes."
(Are the two kinds of intermediate files: 1) the complex shading graph and 2) the final GLSL shading effect?)

Reply:
Makes sense.